src/cdk/table/table.ts

```diff
@@ -403,6 +419,19 @@ export class CdkTable<T> implements AfterContentChecked, CollectionViewer, OnDes
   }
   _multiTemplateDataRows: boolean = false;
 
+  /**
+   * Whether to use a fixed table layout. Enabling this option will enforce consistent column widths
+   * and optimize rendering sticky styles for native tables. No-op for flex tables.
+   */
+  @Input()
+  get fixedLayout(): boolean {
+    return this._fixedLayout;
+  }
+  set fixedLayout(v: boolean) {
+    this._fixedLayout = coerceBooleanProperty(v);
```

Comment:
This may be an extremely uncommon edge case, but would it be correct to reset any values here depending on whether the table is being set one way or the other? E.g. when this changes, should `_forceRecalculateCellWidths` be set to true?